# DNA extraction and 16S amplicon sequencing protocol v1.3: December 15, 2017

***

### 1. Background
This protocol is derived from a “Systematic improvement of amplicon marker gene methods for increased accuracy in microbiome studies” (doi:10.1038/nbt.3601)[www.google.com]. Portions of this protocol have been copied verbatim from the paper and associated protocol (doi:10.1038/protex.2016.030)[www.google.com] but adapted to our materials and equipment. For publication purposes, I think it will be sufficient to cite this protocol.
The purpose of this protocol is to improve accuracy, reduce PCR artifacts such as chimeras, and minimize drop-out of taxa due to mismatches between the amplification or sequencing primers, and to add flexibility in choice of variable region.
This protocol uses a two-step PCR process, where the marker gene of interest is amplified with a pair of adapter-tailed primers in a primary PCR reaction, and sample-specific dual indices and flow cell adapters are added in a subsequent indexing reaction. It could be applied to the following variable regions: V1-V3, V3-V4, V3-V5, V4, V4-V6, and V5-V6 of the 16S ribosomal RNA (rRNA) gene, as well as for variable region V9 of the 18S rRNA gene and the ITS1 and ITS2 regions. The modularity of this design makes it easy to survey new regions of interest, since only a single pair of adapter-tailed locus-targeting primers needs to be synthesized. Primer sets for the primary PCR are listed in Table 8.1 and the indices in Table 8.2.

